graviti/manager/draft.py

```diff
@@ -93,14 +94,29 @@ def close(self) -> None:
         self.state = "CLOSED"
         # TODO: update the draft.updated_at
 
-    def commit(self, title: str, description: str = "") -> None:
+    def commit(self, title: str, description: Optional[str] = None) -> Commit:
```

Comment:
why change the default description to `None`?

Reply:
OpenAPI has a default value for description. Here, we do not need to set default value equal to "". If user do not write description, we don't need to pass this parameter.